fdbserver/ConfigBroadcastFollowerInterface.h

```diff
@@ -0,0 +1,139 @@
+/*
+ * ConfigBroadcastFollower.h
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2018 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "fdbserver/ConfigFollowerInterface.h"
+
+class ConfigClassSet {
+	std::set<Key> classes;
```

Comment:
How is order preserved for the configuration classes other than based on sorted order? For example, if a process has the configuration classes storage/az1, they would be stored in reverse order here.

Reply:
The order is not important here, a worker with config path `storage/az1` will receive all updates to both configuration classes. The order only becomes important when the updates are applied locally.